Tests failing

### Resolvers

Again, the schema from SIMBAD has evolved:

```
./api_resolver_test.py
**********************************************************************
File "/home/peloton/codes/fink-science-portal/tests/./api_resolver_test.py", line 119, in __main__.test_simbad_resolver
Failed example:
    test_simbad_resolver()
Exception raised:
    Traceback (most recent call last):
      File "/home/peloton/anaconda3/lib/python3.9/doctest.py", line 1334, in __run
        exec(compile(example.source, filename, "single",
      File "<doctest __main__.test_simbad_resolver[0]>", line 1, in <module>
        test_simbad_resolver()
      File "/home/peloton/codes/fink-science-portal/tests/./api_resolver_test.py", line 137, in test_simbad_resolver
        assert col in pdf.columns, [col, pdf.columns]
    AssertionError: ['Vel', Index(['name', 'oid', 'oname', 'otype', 'jpos', 'jradeg', 'jdedeg', 'refPos',
           'z', 'MType', 'nrefs'],
          dtype='object')]
**********************************************************************
1 items had failures:
   1 of   1 in __main__.test_simbad_resolver
***Test Failed*** 1 failures.
```

Annoying...

### SSO Candidates

```
./api_ssocand_test.py
**********************************************************************
File "/home/peloton/codes/fink-science-portal/tests/./api_ssocand_test.py", line 117, in __main__.test_time_boundaries
Failed example:
    test_time_boundaries()
Exception raised:
    Traceback (most recent call last):
      File "/home/peloton/anaconda3/lib/python3.9/doctest.py", line 1334, in __run
        exec(compile(example.source, filename, "single",
      File "<doctest __main__.test_time_boundaries[0]>", line 1, in <module>
        test_time_boundaries()
      File "/home/peloton/codes/fink-science-portal/tests/./api_ssocand_test.py", line 123, in test_time_boundaries
        assert not pdf.empty
    AssertionError
**********************************************************************
1 items had failures:
   1 of   1 in __main__.test_time_boundaries
***Test Failed*** 1 failures.
```

Inspecting the data, there are only data (`i:jd`) from 2023... This seems related to the recent reprocessing of SSO Candidates.

--- a/tests/api_resolver_test.py
+++ b/tests/api_resolver_test.py
@@ -131,7 +131,7 @@ def test_simbad_resolver() -> None:
     cols = [
         'name', 'oid', 'oname', 'otype',
         'jpos', 'jradeg', 'jdedeg',
-        'refPos', 'Vel', 'MType', 'nrefs'
+        'refPos', 'MType', 'nrefs'
     ]
     for col in cols:
         assert col in pdf.columns, [col, pdf.columns]

--- a/tests/api_ssocand_test.py
+++ b/tests/api_ssocand_test.py
@@ -117,13 +117,13 @@ def test_time_boundaries() -> None:
     >>> test_time_boundaries()
     """
     pdf = ssocandsearch(
-        kind='lightcurves', start_date='2020-01-01', stop_date='2020-12-31'
+        kind='lightcurves', start_date='2023-01-01', stop_date='2023-12-31'
     )
 
     assert not pdf.empty
 
-    assert np.alltrue(pdf['d:jd'].values >= Time('2020-01-01', format='iso').jd)
-    assert np.alltrue(pdf['d:jd'].values <= Time('2020-12-31', format='iso').jd)
+    assert np.alltrue(pdf['d:jd'].values >= Time('2023-01-01', format='iso').jd)
+    assert np.alltrue(pdf['d:jd'].values <= Time('2023-12-31', format='iso').jd)
 
 
 if __name__ == "__main__":

